ONSdigital/response-operations-social-ui
# SUS055 Hide status and IAC buttons and display message if CE closed

# Motivation and Context
As a: Survey Enquiry Line User
I need: To ensure that a new code cannot be generated, and the case status cannot be changed, if the collection exercise has closed
So that: Respondents do not attempt to access a closed collection exercise erroneously

# What has changed
- Fetch CE end date
- Check date has passed
- Content and front end template
- pass the variable to the template
- Acceptance test

# How to test?
- Run unit tests
- Create collection exercise that has exercise_end in the past
- Check status and IAC buttons are hidden from case details page
- Check content of message show in template
- Check page matches attached design screenshot

# Links
https://trello.com/c/Myki1tNh/323-sus055-uac-cannot-be-generated-if-the-collection-exercise-has-closed-5

# Screenshots
<img width="1280" alt="screen shot 2018-11-07 at 13 24 51" src="https://user-images.githubusercontent.com/43346934/48134034-9f43f880-e290-11e8-8b06-b9e845ab7f33.png">
